python/ray/util/state/state_cli.py

```diff
@@ -353,6 +353,7 @@ def _should_explain(format: AvailableFormat) -> bool:
 @click.argument(
     "id",
     type=str,
+    required=False,
```

Comment:
The error message `Error: Missing argument 'ID'.` is thrown by `click`. We need to set `required` to false to control the error message when an 'ID' is missing.